iacrcc/iacrcc.cls

```diff
@@ -463,6 +463,7 @@ xscale=\@OrigHeightRecip*\@curXheight,transform shape]
                   \hss\@textsuperscript{\normalfont\@thefnmark}}##1}%
       \newpage
       \global\@topnum\z@   % Prevents figures from going at top of page.
+      \renewcommand*{\thefootnote}{\alphalph{\value{footnote}}}%
```

Comment:
doesn't this define it globally for both author footnotes and regular footnotes? It might be preferable to keep these distinct, since the only counter problem we have is for author footnotes.

Reply:
No, I created a test for this in this same pull requests which has an example: test16.